openedx/core/djangoapps/content/block_structure/config/__init__.py

```diff
@@ -10,12 +10,47 @@
 from .models import BlockStructureConfiguration
 
 # Switches
+# .. toggle_name: block_structure.invalidate_cache_on_publish
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: When enabled, the block structure cache is invalidated when changes to
+#   courses are published. Enabling this toggle is useful for non-expiring block structure cache.
+#   It is also useful to enable this when `block_structure.storage_backing_for_cache` is enabled
+#   because it guarantees the storage, containing the storage-backed structures, is updated with
+#   the latest changes.
+# .. toggle_use_cases: opt_in
+# .. toggle_creation_date: 2017-02-23
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/14571
 INVALIDATE_CACHE_ON_PUBLISH = WaffleSwitch(
     "block_structure.invalidate_cache_on_publish", __name__
 )
+
+# .. toggle_name: block_structure.storage_backing_for_cache
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: When enabled, stores block structures in a more permanent storage, like
+#   a database, which provides an additional backup for cache misses, instead of regenerating...
```

Comment:
> The ... at the end was meant to reference the rest of your original sentence. Can you add back the lost text?

I don't know how I missed changing that. Trust me, I'm just as surprised that the `...` is in the text 🤦🏼 

> Could you add the following here, and remove it from the other annotations? If you feel strongly about leaving this text in all the annotations, please update them with this text as well.

Yes, definitely 👍🏼 

I removed it from the other locations, and only kept it for `block_structure.storage_backing_for_cache` because they all depend on it being enabled.

---

Both should be addressed now 👍🏼